code/modules/supermatter/supermatter.dm

```diff
@@ -351,6 +351,15 @@
 */
 
 /obj/machinery/power/supermatter/attackby(obj/item/weapon/W as obj, mob/living/user as mob)
+
+	if(istype(W,/obj/item/weapon/tape_roll))
+		var/obj/item/weapon/tape_roll/tape = W
+		playsound(src, 'sound/items/tape.ogg',25)
+		user.visible_message("<span class='notice'>\The [user] applies duct tape to \the [src], repairing it.</span>")
+		damage -= 5
+		tape.take_charge(user,1)
+		return
```

Comment:
This pretty ridiculous balance-wise. A single role of tape can heal 50 SM damage. Pathetic.

Reply:
of course you of all people would know how to powergame with a feature lmao!!!!!!

Reply:
Tape is honestly one of the weakest tools in the game. With all the recent PRs nerfing tape, I honestly think that we should take a step back and look at ourselves and where the game is coming. I don't want to upset anyone by asking for them to revert their PRs, so I rather write my own code and hopefully come to a compromise.